<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actOpen"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actAdd"/>
    <addaction name="actDelete"/>
    <addaction name="separator"/>
    <addaction name="actUndo"/>
    <addaction name="actRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
